CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+## Contributing
+
+Hi there! We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great.
+
+We accept pull requests for bug fixes and small features where the approach will be simple and straightforward. We'd also love to hear about ideas for new features as issues.
+
+We track issues on our project board [here](https://github.com/OffchainLabs/arbitrum-token-bridge/issues).
+
+Please do:
+
+- Check existing issues to verify that the bug or feature request has not already been submitted.
+- Open an issue if things aren't working as expected.
+- Open an issue to propose a significant change.
+- Open a pull request to fix a bug.
+- If you're contributing for the first time here, try and look for any issues labelled [good first issue](https://github.com/OffchainLabs/arbitrum-token-bridge/labels/good%20first%20issue).
+
+Please avoid:
+
+- Opening pull requests for issues marked `Status: Design Needed`, `Status: To Be Discussed`, or `Status: Need Exploration`.
+
+<!--
+Contributions to this project are [released](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) to the public under the [project's open source license](LICENSE).
+
+Please note that this project is released with a [Contributor Code of Conduct][code-of-conduct]. By participating in this project you agree to abide by its terms. -->
+
+## Instructions to set up the code
+
+Please check our [README section](/README.md) to find detailed instructions about the code setup, as well as running and testing the code locally.
```

Comment:
case-sensitive, README.md is a dead link

Reply:
Updated all `.md` to a consistent uppercase